application-templates/starter/package.json

```diff
@@ -38,6 +38,6 @@
     "@commercetools-frontend/mc-scripts": "15.8.0",
     "@testing-library/react": "9.4.0",
     "enzyme": "3.11.0",
-    "jest": "24.9.0"
+    "jest": "25.1.0"
```

Comment:
Should we consider this a breaking change? Cause the jest peer dep also needs to be bumped.

Reply:
Thanks for raising. I think to me it's not a breaking change. Only node 6 support is dropped and we require node 10/12 in other cases already. All other breaking changes I see are dependent to how people use jest within our peer dependency range. So if somebody imported from `@jest/core` or `jest-cli` and that now changed is not something we can make guarantees about. If their remain to have 24 installed nothing would break for them in anyway. Do you agree or have other concerns about this?

Reply:
Yeah if we support both version it should be fine like this. Let's maybe mention it in the changelog that people can start using jest 25.